Pipes - Amy Cash - Solar System.rb

```diff
@@ -0,0 +1,196 @@
+#Wave Three (Wave One Hash Version follows, but is commented out; Wave Two is encapsulated in Wave Three)
+
+class Planet
+
+  attr_reader :name, :namesake, :type, :year, :distance 
+
+  def initialize(name, namesake, type, year, distance)
+    @name = name
+    @namesake = namesake
+    @type = type
+    @year = year
+    @distance = distance
+  end
+
+  def give_info
+    list =  "#{@name}: \tNamed for: #{@namesake} \tType: #{type} \tYear: #{year} Earth Years \tDistance: \t#{distance} million kilometers."
+    return list
+  end
+
+end
+
+class SolarSystem
+
+  def initialize(list_of_planets)
+    @planets = list_of_planets
+  end
+
+  def add_planet(planet)
+    @planets += planet
+  end
+
+  def print_list
+    list = ""
+    @planets.each_with_index do |planet, index|
+      list += @planets[index].give_info + "\n"
```

Comment:
I like that you use the existing `give_info` method on the `Planet` class rather than formatting the string from scratch.